Libraries/PeriphDrivers/Include/MAX32690/mxc_sys.h

```diff
@@ -204,11 +204,6 @@ typedef enum {
         MXC_V_GCR_CLKCTRL_SYSCLK_SEL_EXTCLK /**< Use the external system clock input */
 } mxc_sys_system_clock_t;
 
-typedef enum {
-    MXC_SYS_RISCV_CLOCK_ISO, /**< Select the Internal Secondary Oscillator (ISO) as the RISCV clock source */
-    MXC_SYS_RISCV_CLOCK_PCLK /**< Select the Advanced Peripheral Bus (APB) clock as the RISCV clock source */
-} mxc_sys_riscv_clock_t;
```

Comment:
Your choice to include the deprecated attribute to this enum. The downside is the deprecated message will pop up anytime "mxc_sys.h" is included in a file, and many libraries within the SDK include it... whereas adding the deprecated attribute only to the function should throw the warning only when the function is used.